ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -195,6 +198,25 @@ public ConsistencyCheckerTool(ClusterMap clusterMap, StoreKeyFactory storeKeyFac
     return new Pair<>(success, resultsByReplica.getSecond());
   }
 
+  /**
+   * Will examine the index files in the File replicas for store keys
+   * and will then convert these store keys and return the
+   * conversion map
+   * @param replicas replicas that should contain index files that will be
+   *                 parsed
+   * @return conversion map of old store keys and new store keys
+   * @throws Exception
+   */
+  public Map<StoreKey, StoreKey> convertBlobIds(File[] replicas) throws Exception {
+    Pair<Boolean, Map<File, DumpIndexTool.IndexProcessingResults>> resultsByReplica =
+        getIndexProcessingResults(replicas);
+    boolean success = resultsByReplica.getFirst();
```

Comment:
minor: unnecessary variable?

Reply:
Could directly put it in, but as it is it's useful as a bit of self-documenting code